app/Jobs/DeploymentQueueJob.php

```diff
@@ -21,57 +19,48 @@ class DeploymentQueueJob extends Job implements ShouldQueue
     protected $deploy;
 
     /**
-     * Create a new job instance.
-     *
-     * @return void
-     */
+    * Create a new job instance.
+    *
```

Comment:
bad indentation, docblockr should be like this: https://github.com/firework/deployer/pull/28/files#diff-204cb905769b13dff7f0a867415140a4R10